.gitignore

```diff
@@ -16,3 +16,4 @@
 /coverage
 /.env
 /public/uploads
+/public/uploads/user/avatar/*/example.jpg
```

Comment:
You can probably ignore the entire uploads directory. Uploads are data generated once the app is running, not part of the app codebase:

```ruby
/public/uploads
```